crowdin/messages.json

```diff
@@ -2523,6 +2521,7 @@
   "-610649986": "Crypto pairs",
   "-1593593334": "Commodities Basket",
   "-1479581544": "Forex Basket",
+  "-785091863": "Open a real account, make a deposit, and start trading commodities and other markets. ",
```

Comment:
you two of the same string, one with extra space at the end. 
please check which one is the correct one and remove the other one, thanks :pray: